lib/fluent/plugin/filter_process_ucs_syslog.rb

```diff
@@ -240,6 +242,31 @@ def getToken(host)
       return token
     end
 
+    def updateEtcd(record)
+      sourceIp = record[ucsHostNameKey]
+
+      uri = URI.parse("http://etcd:2379/v2/keys/#{sourceIp}")
```

Comment:
It seems like this is $CONTAINER_NAME:$PORT, could you move them to constants?

Reply:
done